packages/apify/src/actor.ts

```diff
@@ -1553,20 +1553,25 @@ export interface MainOptions extends ExitOptions, InitOptions {}
  */
 export interface ApifyEnv {
     /**
-     * ID of the actor (ACTOR_ID)
+     * ID of the actor (APIFY_ACTOR_ID)
```

Comment:
please revert those, i believe we want to keep the plain `ACTOR_` env vars (check the git history for more details)

Reply:
Yeah, took a look at previous PRs but still not sure what is the thumb rule for delienating these as there is a lot of overlap. Are ACTOR_ specific to actor and APIFY_ to platform/user acc? If so, I should then replace all APIFY_ prefixes if there is an equivalent with ACTOR_, right?

Reply:
i believe everything that has `ACTOR_` variant should use that, not sure if there are some that don't

Reply:
For context, originally the env vars were called `APIFY_XXX`, but with the introduction of the Actor philosophy, we wanted to de-couple Actors from Apify, and hence redefined the env vars as non-Apify `ACTOR_XXX` instead (see [here](https://github.com/apify/actor-specs#environment-variables)). The Apify SDK should use those whenever possible, the old `APIFY_XXX` env vars are kept on the platform just for backwards compatability.